_summaries/gropp2020implicit_2.md

```diff
@@ -1,19 +1,85 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: gropp2020implicit
+author: # samatharhay
+score: # 7
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+The core idea of the paper was to introduce a new paradigm built on implicit geometric regularization for computing high fidelity implicit neural representations directly from raw data scans.
+
 * How is it realized (technically)?
+
+Given an input cloud X = {x_i}iEI C R^3 with or without normal data, N = {n_i} iEI C R^3, the goal is to compute the parameters \theta of a multi-layer perceptron (MLP) f(x;\theta), where f has the shape R^3 x R^m \rightarrow R, such that it approximates a signed distance function to a plausible surface M defined by X and N.
+
+To help learn these parameters the authors propose the loss function:
+
+![image of function 2](gropp2020implicit_2a.png)
+
+where \lambda > 0 is a hyperparameter, ||.|| = ||.||2 (the euclidean 2-norm) and 
+
+![image of function 3](gropp2020implicit_2b.png)
+
+which encourages f to vanish on X.
+
+The second term in the first equation is called the Eikonal term, which encourages the gradients to be of unit 2-norm.
+
+Optimizing the loss function using SGD, or a variant of SGD) results in solutions that are close to a signed distance function with a smooth and suprising ly plausible zero level set. 
+
+![insert image 1](gropp2020implicit_2c.png)
+
+Instead of using numerical estimes of the gradient when incorporating them into the loss, the authers used:
+
+![equation 4](gropp2020implicit_2d.png)
+
+Comparing this method to other popular methods:
+
+Regular Grid Volumetric Function (RGVF):
+* implicit function is defined only at grid points, requiring an interpolation scheme to extend it to interior of cells
+* It requires cubic-size grid and is not data-dependent meaning it does not adhere to the specific geometry of the shapes one wishes to approximate
+* a version of the Eikonal regularization term has been used as a normalization term, as apposed to this work which uses it alone as regularizing the zero level set.
+* gradients are computed using finite differences or via some fixed basis function.
+
+Neural parametric surfaces:
+* finding a consistent atlas covering of the surface can be challenging.
+* the shape can be easily sampled.
+* challenging to produce a set of perfectly overlapping charts, a property that holds by construction for implicit representations.
```

Comment:
I don't think the related works are necessary, but you can mention the parts you think are important that this paper differs from others.